data/user_data_fetch_github.ps2

```diff
@@ -0,0 +1,17 @@
+<powershell>
+# Fetch Windows User Data from GitHub
+
+$HUDL_DIRECTORY = "c:\\hudl"
+$USER_DATA_URI = "https://s3.amazonaws.com/hudl-chef-artifacts/windows/user_data_base.ps2"
+$USER_DATA_FILE_PATH = "c:\\hudl\\user_data_base.ps1"
+
+if(!(Test-Path -Path $HUDL_DIRECTORY )){
+    New-Item -ItemType directory -Path $HUDL_DIRECTORY
+}
+
+Write-Output "Fetching user data from GitHub"
+Invoke-WebRequest $USER_DATA_URI -OutFile $USER_DATA_FILE_PATH
```

Comment:
You might need to add -UseBasicParsing to this.  I'd guess that 'non-basic' parsing probably isn't possible from the user account that runs this script.